SELF_ASSESSMENT.md

```diff
@@ -233,7 +233,56 @@ Automated testing is integrated into most repositories via GitHub Actions. These
   * Outbound. How do you communicate with your users? (e.g. flibble-announce@
     mailing list)-->
 
+<details open>
+<summary><b>Internal</b>: How to communicate with your team mates?</summary>
+<!--All you need is a blank line-->
+
+| Communication Channel                                                                                                                            | Usage                                                                                                                                                       |
+| ------------------------------------------------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Slack](https://cloud-native.slack.com/archives/C06F3PEPNBW)                                                                                     | Real-time chats and team collaboration.                                                                                                                     |
+| [Google Group](https://groups.google.com/g/oscal-compass)                                                                                        | Formal email communications. Address is oscal-compass@googlegroups.com.                                                                                     |
```

Comment:
I don't thing google groups is used to communicate internally between teams. This is more for public announcements.